apps/extension/package.json

```diff
@@ -20,7 +20,7 @@
     "@bufbuild/protobuf": "^1.8.0",
     "@connectrpc/connect": "^1.4.0",
     "@connectrpc/connect-web": "^1.4.0",
-    "@penumbra-labs/registry": "4.0.0",
+    "@penumbra-labs/registry": "^4.1.0",
```

Comment:
Noticed that this package is published by the penumbra-labs organization, but is in organization's prax-wallet repository. I'm assuming this is intentional